doc/sphinx/io.rst

```diff
@@ -164,7 +166,10 @@ to the :class:`~espressomd.io.writer.h5md.H5md` class. Currently available:
     - ``write_mass``: particle masses
 
     - ``write_ordered``: if particles should be written ordered according to their
-      id (implies serial write).
+      id (implies serial write)
+
+    - ``unit_system``: optionally, physical units for time, mass, length and
```

Comment:
what about other SI base units that are defined in the [H5MD documentation](https://nongnu.org/h5md/modules/units.html)?

Reply:
The other SI units (current, temperature, moles, candela) do not seem to apply for espresso particles. The per-particle temperature is more of a trick for tailored Langevin dynamics. Same thing for derived units. Torque could have been relevant, but isn't defined.